src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,49 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    public static final int MAX_AGE = 100;
+    public static final int MIN_AGE = 18;
+    public static final int MIN_LOGIN_LENGTH = 6;
+    public static final int MIN_PASSWORD_LENGTH = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
-    public User register(User user) {
-        return null;
+    public User register(User user) throws RegistrationUserException {
```

Comment:
you should not use `throws` section for Runtime children exceptions. fix all cases in the code please.